pkg/monitor/monitor/util.go

```diff
@@ -1045,12 +1049,12 @@ func getMonitorStatefulSetSkeleton(sa *core.ServiceAccount, monitor *v1alpha1.Ti
 				Type: apps.RollingUpdateStatefulSetStrategyType,
 			},
 			Selector: &meta.LabelSelector{
-				MatchLabels: buildTidbMonitorLabel(monitor.Name),
+				MatchLabels: stsLabels,
 			},
 			Template: core.PodTemplateSpec{
 				ObjectMeta: meta.ObjectMeta{
-					Labels:      buildTidbMonitorLabel(monitor.Name),
-					Annotations: util.CopyStringMap(monitor.Spec.Annotations),
+					Labels:      podLabels,
```

Comment:
Please also support the labels and annotations for the services created for Tidbmonitor。

Reply:
https://github.com/pingcap/tidb-operator/pull/4029/commits/3ea7b432dc9678acd1f63de8d00b47553b4ddf7a